### ISSUE TITLE
For pm-gpu, update cudatoolkit version from 11.5 to 11.7

### ISSUE THREAD

**Issue by ndkeen** [E3SM-Project/E3SM#5314]
I wanted to update cudatoolkit version on pm-gpu from 11.5 to the current system default of 11.7.
The only reason for update is to keep up with system defaults.

I don't think there are any major changes between the versions.
https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html

At one point, there was a failing test using 11.7, but that is no longer the case.
https://github.com/E3SM-Project/scream/issues/1896

**Comment by ndkeen** [comment]
Testing this in SCREAM repo, I ran 2 cases that were the same except changing `cudatoolkit/11.5` to `cudatoolkit/11.7`.
The case is ne120 and runs for 3 months.  I'm only outputting one variable which is the min T. The output files are different from cprnc, but it's not clear how to tell when exactly the run diverges from BFB (if it even started BFB). Not that we expect a diff version of cuda to yield BFB.  We used to have output of floating values in atm.log.

`/pscratch/sd/n/ndk/e3sm_scratch/pm-gpu/se47-dec5/f120.F2010-SCREAMv1.ne120pg2_r0125_oRRS18to6v3.se47-dec5.gnugpu.3m.n032a4xX.Tmidout.ctk115`

compared to:

`/pscratch/sd/n/ndk/e3sm_scratch/pm-gpu/se47-dec5/f120.F2010-SCREAMv1.ne120pg2_r0125_oRRS18to6v3.se47-dec5.gnugpu.3m.n032a4xX.Tmidout.ctk117`

There was no measurable difference in the performance.
